client/signup/steps/woocommerce-install/confirm/index.tsx

```diff
@@ -114,9 +115,21 @@ export default function Confirm( props: WooCommerceInstallProps ): ReactElement
 							onClick={ () => {
 								dispatch( submitSignupStep( { stepName: 'confirm' }, { siteConfirmed: siteId } ) );
 								if ( siteUpgrading.required ) {
+									dispatch(
+										recordTracksEvent( 'calypso_woocommerce_dashboard_confirm_submit', {
```

Comment:
I'm seeing this in my console, but from reading testing instructions, should it be named `calypso_woocommerce_dashboard_upgrade_required`?

Reply:
same here, can confirm!

Reply:
I think we just need clarification on the name and this one is ready to merge :) !

Reply:
> I'm seeing this in my console, but from reading testing instructions, should it be named `calypso_woocommerce_dashboard_upgrade_required`?

Sorry just me forgetting to update the testing instructions!